plugins/nodes/src/js/columns/NodesTableTypeColumn.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { compareValues } from "#PLUGINS/nodes/src/js/utils/compareValues";
+import Node from "#SRC/js/structs/Node";
+// TODO: DCOS-39079
```

Comment:
Is it still the issue? Jira has been resolved https://jira.mesosphere.com/browse/DCOS-39079